builder/Dockerfile

```diff
@@ -2,28 +2,11 @@ FROM golang:1.19.6-bullseye as builder
 
 ENV GOPRIVATE="github.com/companieshouse"
 
-RUN apt-get update && \
-    apt-get install -y --allow-downgrades --no-install-recommends\
-    git=1:2.30.2-1 && \
-    rm -rf /var/lib/apt/lists/*
+ADD ./builder/lib/build /lib/
```

Comment:
As suggested by Joel, maybe copy to /bin instead? If my other suggestion is accepted, this would be:
```
ADD ./builder/bin/go_builder_for_docker /bin/
```